docs/python.md

```diff
@@ -0,0 +1,28 @@
+# Python Client
+
+A python library for interacting with the Limit Order API.
+
+The library is currently tested against Python versions 2.7, 3.4, 3.5, and 3.6
+
+- [GitHub](https://github.com/dydxprotocol/dydx-python)
+- [PyPi](https://pypi.org/project/dydx-python)
+
+## Install
+Install with `pip`:
+```
+pip install dydx-python
+```
+
+## Example Usage
+
+```python
```

Comment:
When I run this locally, it doesn't syntax highlight this, I'm not sure why

Reply:
yeah not sure either